fixtures/dom/src/components/fixtures/number-inputs/index.js

```diff
@@ -33,6 +33,29 @@ function NumberInputs() {
         </p>
       </TestCase>
 
+      <TestCase
+        title="Backspacing in Shadow DOM"
+        description="The decimal place should not be lost in a Shadow DOM element">
+        <TestCase.Steps>
+          <li>Create number input inside a Shadow DOM element</li>
+          <li>Type "3.1"</li>
+          <li>Press backspace, eliminating the "1"</li>
+        </TestCase.Steps>
+
+        <TestCase.ExpectedResult>
+          The field should read "3.", preserving the decimal place
+        </TestCase.ExpectedResult>
+
+        <NumberTestCase />
```

Comment:
The formatting is perfect here, but it doesn't look like this attaches to a Shadow DOM. What is the best way to do that?

Reply:
I’m not entirely sure how to deal with a Shadow DOM in the test fixtures. I’ll look into it.